pkg/util/utils_test.go

```diff
@@ -17,3 +18,54 @@ func TestStringInSlice(t *testing.T) {
 	// string is not in empty slice
 	assert.False(t, StringInSlice("one", []string{}))
 }
+
+func TestParseChanges(t *testing.T) {
+	// CMD=/bin/sh
+	_, vals, err := ParseChanges("CMD=/bin/sh")
+	assert.EqualValues(t, []string{"/bin/sh"}, vals)
+	assert.NoError(t, err)
+	// CMD [/bin/sh]
+	_, vals, err = ParseChanges("CMD [/bin/sh]")
+	assert.EqualValues(t, []string{"/bin/sh"}, vals)
+	assert.NoError(t, err)
+
+	// CMD ["/bin/sh"]
+	_, vals, err = ParseChanges(`CMD ["/bin/sh"]`)
+	assert.EqualValues(t, []string{`"/bin/sh"`}, vals)
+	assert.NoError(t, err)
+
+	// CMD ["/bin/sh", "-c", "ls"]
+	_, vals, err = ParseChanges(`CMD ["/bin/sh","c","ls"]`)
```

Comment:
Can we add tests with spaces between items `["a", "b"]` and others with a commas in the strings `["/bin/foo", "--string-arg", "' some sentence\, with a comma'"]`?

Reply:
comma is not checked for invalid value